src/formatters/base.js

```diff
@@ -182,11 +182,7 @@ class BaseFormatter {
       if (arrayKeys && key === '_t') {
         continue;
       }
-      const leftKey = arrayKeys
-        ? typeof key === 'number'
-          ? key
-          : parseInt(trimUnderscore(key), 10)
-        : key;
+      const leftKey = arrayKeys ? parseInt(trimUnderscore(key), 10) : key;
```

Comment:
`key` cannot be a number because all of the values pushed to the `keys` are strings. `keys` and `key` are both local variables so there is no concern of it being used differently by users.